<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CE2_MNARF</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_AbrirNetlist">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir Netlist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Programa para Análise Nodal Modificada no Estado Permanente Senoidal</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pte_JanelaTexto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Fechar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_frequenciaInicial">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequência Inicial</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_frequenciaFinal">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequência Final</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_numeroPontos">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de Pontos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_frequenciaInicial">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_frequenciaFinal">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_numeroPontos">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_linear">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>330</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_oitava">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oitava</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_decada">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>390</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Década</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>152</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cb_MostrarEstampas">
       <property name="text">
        <string>Mostrar as Estampas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_MostrarResultados">
       <property name="text">
        <string>Mostrar os Resultados</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_ignorarComando">
       <property name="text">
        <string>Ignorar Linha de Comando</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
